corehq/apps/reports/dispatcher.py

```diff
@@ -184,6 +193,15 @@ def _rendering_pattern(cls):
 
     @classmethod
     def pattern(cls):
+        # grep help:
+        # ^(json/)?(?P<report_slug>[\w_]+)/$
+        # ^(async/)?(?P<report_slug>[\w_]+)/$
+        # ^(filters/)?(?P<report_slug>[\w_]+)/$
+        # ^(export/)?(?P<report_slug>[\w_]+)/$
+        # ^(mobile/)?(?P<report_slug>[\w_]+)/$
+        # ^(email/)?(?P<report_slug>[\w_]+)/$
+        # ^(partial/)?(?P<report_slug>[\w_]+)/$
+        # ^(print/)?(?P<report_slug>[\w_]+)/$
         return r'^({renderings}/)?(?P<report_slug>[\w_]+)/$'.format(renderings=cls._rendering_pattern())
```

Comment:
I think it would be better to write a short documentation where this would be relevant rather than adding routes in the code. If I am looking for a view for a url, I don't see myself searching for the url regex in the code and finding this anyway.

Reply:
How do you generally find view URLs?
I normally search for a keyword in the URL, along with the app name that I think it would be in.

## Before
![Screenshot from 2022-01-25 09-59-09](https://user-images.githubusercontent.com/146896/151001336-d408b587-96c8-42b3-901d-a18184bdc5f9.png)
## After
![Screenshot from 2022-01-25 09-59-43](https://user-images.githubusercontent.com/146896/151001351-7fea62d3-b62c-460e-ba7f-b5682b2016c8.png)

I'm not trying to solve this problem for every single workflow, just those that use grep.

Reply:
> How do you generally find view URLs?
I normally search for a keyword in the URL, along with the app name that I think it would be in.

Quite similar. I look for the keyword in just the files ending with name `urls.py` to limit my search.

Reply:
I usually use `./manage.py show_urls | grep mykeyword` (from django-extensions).  Whenever I've tried that with reports I just sigh and open the dispatcher code.  I wonder how hard it'd be to dynamically register these URLs with Django (could even do it for each report), which I think would "fix" that workflow (clearly outside the scope of this PR)